docs/fmudesign.rst

```diff
@@ -315,43 +236,3 @@ monte carlo sensitivities.
 
     # designtable is a pandas DataFrame with summary of the design in the designmatrix,
     # i.e. it will contain realisation number, senstype and senscase for each sensitivity
-
-Example: calculating one tornadotable
-""""""""""""""""""""""""""""""""""""""
-
-Using calc_tornadoplot with a 'designsummary' and a resultfile as
-input, and calculating statistics to visualize in a tornado plot for a
-given choice of SELECTOR (e.g. ZONE:'Nansen') and RESPONSE
-(e.g. STOIIP_OIL). The reference is usually the mean of the
-realizations in the "seed sensitivity", but it can also be specified
-as a single realisation number, e.g. if you have a reference case in
-realization 0. Statistics showing the difference to the reference can
-be calculated as absolute values, or as percentages. You could also
-choose to exclude from the plot, sensitivities that are smaller than
-the seed sensitivity P10/P90.
-
-.. code-block:: python
-
-    #!/usr/bin/env python
-    # -*- coding: utf-8 -*-
-
```

Comment:
This example belongs to the code that will be remaining in repository and can be kept: the rest can be deleted.

Reply:
I think the rest of the changes looks ok.

Reply:
like so?